pkg/gardenadm/botanist/etcd.go

```diff
@@ -50,7 +50,7 @@ func (b *GardenadmBotanist) DeployEtcdDruid(ctx context.Context) error {
 
 	deployer, err := sharedcomponent.NewEtcdDruid(
 		b.SeedClientSet.Client(),
```

Comment:
Should we store the instance of the druid component on `GardenadmBotanist.Components` just like the runtime resource-manager?

Reply:
We can do that once we need it - for now, there is no other use of the druid component. We also have already other components that we don't store (for the same reason): https://github.com/gardener/gardener/blob/b752a9f1c3e08c28202f48ee058222a0a1a0a9c8/pkg/gardenadm/botanist/etcd.go#L94-L103